### ISSUE TITLE
bib2gls breaks hypergroup styles

### ISSUE THREAD

**Issue by nlct** [nlct/bib2gls#14]
MWE:
```latex
\documentclass{article}

\usepackage[colorlinks]{hyperref}
\usepackage[record]{glossaries-extra}

\GlsXtrLoadResources[src={example-glossaries-brief},selection=all]

\begin{document}
\printunsrtglossary[style={listhypergroup}]
\end{document}
```

`bib2gls` adds the following code to the `.glstex` file:
```latex
\ifdef\@glsnavhypertarget
{
  \ifdef\bibglsorgnavhypertarget
  {}
  {
    \let\bibglsorgnavhypertarget\@glsnavhypertarget
  }
  \renewcommand*{\@glsnavhypertarget}[3]{%
    \ifcsdef{@gls@hypergrouplist@#1}%
    {\@glstarget{\glsnavhyperlinkname{#1}{#2}}{#3}}%
    {\bibglsorgnavhypertarget{#1}{#2}{#3}}%
  }
  \providecommand{\bibglshypergroup}{\@gls@hypergroup}
}
{
  \providecommand{\bibglshypergroup}[2]{}
}
```
The original aim was to by-pass the need for an additional LaTeX call by obtaining the group hyper targets from the group information that `bib2gls` has obtained. Unfortunately it wasn't fully implemented and can't be properly implemented as settings that `bib2gls` is unaware of may create different targets. The above code disables the default way of obtaining the hyper targets (via the `.aux` file) which results in destabilising the document build. Alternate LaTeX calls produce the warning:
```
Package glossaries Warning: Navigation panel has changed. Rerun LaTeX.
```

**Comment by nlct** [comment]
The problem occurs when `bib2gls` doesn't know the glossary type.

**Comment by nlct** [comment]
Fixed in commit 7a3fb23
